### PR TITLE
#220: Made main page footer component

### PR DESCRIPTION
**Pull request by afijal**
I've decided not to split it into smaller components as those elements won't be reusable

### CODE REVIEW

File: src/components/Home/components/Footer/styles.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.Benetech {
+    flex: 0 0 100px;
+}
+
+.BenetechImg {
+    float: right;
+    margin: 0px 10px 0px 25px; 
+    background-color: #FFF; 
+    padding: 5px;
+}
```
Comment (pawelcieszko) [inline src/components/Home/components/Footer/styles.css]: add new line

Comment (dserkowski) [inline src/components/Home/components/Footer/styles.css]: Add the new line character

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* style to use if only the screen reader user should hear this */
+.SROnly {
+    clip: rect(1px, 1px, 1px, 1px);
+    position: absolute !important;
+    white-space: nowrap;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+  }
```
Comment (pawelcieszko) [inline src/styles.css]: add new line

File: src/components/Home/components/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import footer from './styles.css';
+import styles from '../../../../styles.css';
+
+
```
Comment (pawelcieszko) [inline src/components/Home/components/Footer/index.js]: remove undesired line

File: src/components/Home/components/Footer/styles.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.footer {
+    flex: 0 0 50px;
+    margin-bottom: 10px; 
+    background-color: #001F3F; 
+    border-top: 10px solid #001F3F; 
+    display: flex; 
+    border-bottom: 10px solid #001F3F;
+  }
+
+.EdGov {
```
Comment (pawelcieszko) [inline src/components/Home/components/Footer/styles.css]: shouldn't the css names start lowercase? not sure, just asking

File: src/components/Home/components/Footer/index.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import footer from './styles.css';
+import styles from '../../../../styles.css';
+
```
Comment (oskarhinc) [inline src/components/Home/components/Footer/index.js]: Duplicated empty lines

File: src/styles.css (hunk 2 above)
Comment (dserkowski) [inline src/styles.css]: Add the new line character

File: src/components/Home/components/Footer/index.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+export default class MainPageFooter extends React.Component {
+    render() {
+      return (
+      <div id="footer" className="{footer.footer}">
```
Comment (oskarhinc) [inline src/components/Home/components/Footer/index.js]: Adjust indent here

File: src/components/Home/components/Footer/index.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+              <div className={footer.EdGov}>
+                  <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html" target="_blank">
+                      <img className={footer.EdGovImg}
+                           src="images/IDEA-logo.gif" alt="IDEA Logo" height="50"/>
+                  </a>
+              </div>
+              <div className={footer.FooterText}>
+                  The <a href="http://diagramcenter.org/" target="_blank">DIAGRAM Center</a> is a <a
+                      href="http://www.benetech.org/"
+                      target="_blank">Benetech </a>
```
Comment (dserkowski) [inline src/components/Home/components/Footer/index.js]: I think that this space character is not necessary

File: src/index.js [hunk 8]
```diff
@@ -1,13 +1,8 @@
...
+import MainPageFooter from './components/Home/components/Footer/index.js';
 
-const Index = () => {
-  return ( 
-        <div>
-          Hello
-        </div>
-  );
-};
+ReactDOM.render(<MainPageFooter />, document.getElementById("footerTestPlaceHolder"));
```
Comment (dserkowski) [inline src/index.js]: Should we leave the space character after a tag name?

File: src/index.html [hunk 9]
```diff
@@ -226,35 +226,7 @@ <h3 class="sr-only">Clean up and add new step</h3>
...
-                2012-2017, Beneficent Technology, Inc.
-            </div>
-            <div style="flex: 0 0 100px;">
-                <a href="https://www.benetech.org/" target="_blank">
-                    <img style="float: right;margin: 0px 10px 0px 25px; background-color: #FFF; padding: 5px;"
-                         src="images/benetech_logo_transparent.gif" alt="Benetech Logo" height="50">
-                </a>
-            </div>
-        </footer>
+    <div id="footerTestPlaceHolder">
```
Comment (oskarhinc) [inline src/index.html]: Why not just "footer"?

Comment (dserkowski) [reply src/index.html]: I think it is because it is the temporary solution. Maybe footerTestPlaceHolder would be better...

Comment (oskarhinc) [reply src/index.html]: footerPlaceHolder sounds good as well for me

**Comment by afijal** [comment]
@dserkowski @pawelcieszko  Thanks, I've corrected mentioned issues

File: src/index.html (hunk 9 above)
Comment (afijal) [reply src/index.html]: exactly, fine then, I've removed "test"

File: src/components/Home/components/Footer/styles.css (hunk 1 above)
Comment (afijal) [reply src/components/Home/components/Footer/styles.css]: k

File: src/styles.css (hunk 2 above)
Comment (afijal) [reply src/styles.css]: k

File: src/components/Home/components/Footer/styles.css (hunk 1 above)
Comment (afijal) [reply src/components/Home/components/Footer/styles.css]: k

File: src/styles.css (hunk 2 above)
Comment (afijal) [reply src/styles.css]: k

File: src/components/Home/components/Footer/index.js (hunk 5 above)
Comment (afijal) [reply src/components/Home/components/Footer/index.js]: k

File: src/index.js [hunk 10]
```diff
@@ -1,13 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import MainPageFooter from './components/Home/components/Footer/index.js';
```
Comment (oskarhinc) [inline src/index.js]: /index.js is not needed here.

**Pull request merged**